csrankings-y.csv

```diff
@@ -291,6 +291,7 @@ Yiming Hu,University of Cincinnati,http://www.ece.uc.edu/~yhu,NOSCHOLARPAGE
 Yiming Yang,Carnegie Mellon University,http://www.cs.cmu.edu/~./yiming,MlZq4XwAAAAJ
 Yin Pan,Rochester Institute of Technology,https://www.rit.edu/gccis/computingsecurity/people/yin-pan,SHJ98SkAAAAJ
 Yin Tat Lee,University of Washington,http://yintat.com,nwHfwCIAAAAJ
+Yin Yang 0002,Clemson University,https://yangzzzy.github.io,-z2_nggAAAAJ

```

Comment:
Not listed here: https://www.clemson.edu/cecas/departments/computing/people/faculty/computer_science.html

Reply:
Listed here: https://www.clemson.edu/cecas/departments/computing/people/faculty/index.html